cmd/maya-agent/app/exporter/collector/collector.go

```diff
@@ -15,6 +15,16 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 )
 
+const (
```

Comment:
move all constants to `pkg/util/constants.go`, these constants had been used in `volume_stats.go` file too.
So better to import them.